File: app/src/main/java/dahoum/wales/access_app/models/Place.java
package dahoum.wales.access_app.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Place {
    @SerializedName("id")
    @Expose
    private String id;
    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("type")
    @Expose
    private String type;
    @SerializedName("image")
    @Expose
    private String image;
    @SerializedName("description")
    @Expose
    private String description;
    @SerializedName("www")
    @Expose
    private String www;
    @SerializedName("location")
    @Expose
    private String location;
    @SerializedName("isFavourite")
    @Expose
    private Boolean isFavourite;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public String getImage() {
        return image;
    }

    public void setImage(String image) {
        this.image = image;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getWww() {
        return www;
    }

    public void setWww(String www) {
        this.www = www;
    }

    public String getLocation() {
        return location;
    }

    public void setLocation(String location) {
        this.location = location;
    }

    public Boolean getIsFavourite() {
        return isFavourite;
    }

    public void setIsFavourite(Boolean isFavourite) {
        this.isFavourite = isFavourite;
    }
}

File: resources/src/main/java/app/downloadaccess/resources/models/Place.kt
package app.downloadaccess.resources.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Place : Serializable {
    @SerializedName("id")
    @Expose
    var id: String? = null

    @SerializedName("typeId")
    @Expose
    var typeId = 0

    @SerializedName("name")
    @Expose
    var name: String? = null

    @SerializedName("type")
    @Expose
    var type: String? = null

    @SerializedName("image")
    @Expose
    private var image: String? = null

    @SerializedName("description")
    @Expose
    var description: String? = null

    @SerializedName("www")
    @Expose
    var www: String? = null

    @SerializedName("address")
    @Expose
    var address: String? = null

    @SerializedName("location")
    @Expose
    var location: String? = null

    @SerializedName("isFavourite")
    @Expose
    var isFavourite: Boolean? = null

    @SerializedName("approved")
    @Expose
    var approved: Boolean? = false

    fun     setImage(image: String?) {
        this.image = image
    }

}